internal/installation/stop.go

```diff
@@ -78,14 +78,18 @@ func stopSvc(installPath string) error {
 			continue
 		}
 
-		exe, err := p.Exe()
-		if err != nil {
-			logging.Error("Could not get executable path for process %s, error: %v", n, err)
-			continue
-		}
+		if n == constants.ServiceCommandName {
+			exe, err := p.Exe()
+			if err != nil {
+				logging.Error("Could not get executable path for state-svc process, error: %v", err)
+				continue
+			}
+
+			if !strings.Contains(strings.ToLower(exe), "activestate") {
```

Comment:
Come to think of it, let's log this as an error. I'd like to know if this happens.